### DISCUSSION TITLE
Creating a new custom abp-taghelper, is it possible?

### DISCUSSION THREAD

**Discussion by rodrigomarcell** [abpframework/abp#11994] [Q&A] [answered]
Does anyone know how do I create my own abp-taghelper customized on abp?

**Comment by maliming** [comment, marked as answer]
You can see 
https://github.com/abpframework/abp/tree/dev/framework/src/Volo.Abp.AspNetCore.Mvc.UI.Bootstrap/TagHelpers
https://github.com/abpframework/abp/pull/6755
